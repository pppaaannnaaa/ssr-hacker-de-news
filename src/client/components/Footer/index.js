import React from 'react';

export default () => {
  return (
    <footer className="page-footer blue" role="navigation" aria-label="News de Hacker copyright">
      <div className="footer-copyright">
        <div className="container">Â© 2020 SSR News de Hacker</div>
      </div>
    </footer>
  );
};
