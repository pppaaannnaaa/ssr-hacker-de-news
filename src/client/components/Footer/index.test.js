import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Footer from './index';

describe('<Footer />', () => {
  let props;

  beforeEach(() => {
    props = {};
  });

  afterEach(cleanup);

  test('Renders <Footer /> component correctly', () => {
    const { container, getByText } = render(<Footer />);
    expect(container).toMatchSnapshot();
    expect(getByText('Â© 2020 SSR News de Hacker')).toBeInTheDocument();
  });
});
