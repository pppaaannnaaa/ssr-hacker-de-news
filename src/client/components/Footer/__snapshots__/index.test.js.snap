// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Renders <Footer /> component correctly 1`] = `
<div>
  <footer
    aria-label="News de Hacker copyright"
    class="page-footer blue"
    role="navigation"
  >
    <div
      class="footer-copyright"
    >
      <div
        class="container"
      >
        Â© 2020 SSR News de Hacker
      </div>
    </div>
  </footer>
</div>
`;
